config/webpack.prod.js

```diff
@@ -33,6 +33,7 @@ const OfflinePlugin = require('offline-plugin');
 const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
 const FABRIC8_FORGE_API_URL = process.env.FABRIC8_FORGE_API_URL;
 const FABRIC8_WIT_API_URL = process.env.FABRIC8_WIT_API_URL;
+const REALM = process.env.REALM || 'fabric8';
```

Comment:
The ENV var should be set in the build config the way "FABRIC8_BRANDING=openshiftio" is on https://github.com/fabric8io/fabric8-ui/blob/master/cico_build_deploy.sh#L27